### DISCUSSION TITLE
MiniOS

### DISCUSSION THREAD

**Discussion by bewithage** [minios-linux/minios-live#24] [Q&A]
How long should it take to install MiniOS? Is there a message that says the install is done. It's been about an hour since I started. There is still an icon on the desktop to install MiniOS.

**Comment by egordev0** [comment]
uhh.... What

**Reply by bewithage** [reply]
I can boot to MiniOS from the USB stick. All that works but when I click on the desktop icon to install MiniOS nothing happens. I even left the computer running overnight and still MiniOS not installed on the hard disk.
